projects/storefrontlib/cms-components/cart/add-to-cart/add-to-cart.component.ts

```diff
@@ -14,6 +14,7 @@ import {
   isNotNullable,
   Product,
 } from '@spartacus/core';
+import { AddedToCartToastConfig } from 'feature-libs/cart/added-to-cart-toast/added-to-cart-toast-config';
```

Comment:
I am not sure how about this line, can we use feature lib element in storefrontlib, it is lazy loading so it is probably problematic here.